.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_SMART_PROXY: '0'
```

Comment:
Doesn't an empty string work just as well?

Also, you can set `BUNDLE_WITHOUT: development`

Reply:
That would not work IMO, the smart-proxy is not in development and on contrary, we want to install some development dependencies.